wcivf/apps/elections/templates/elections/election_view.html

```diff
@@ -46,9 +45,11 @@ <h3>{{ object.nice_election_name }}</h3>
                         {% endif %}
 
                     {% else %}
-                        {% if not object.in_past and not election.slug == 'parl.2017-06-08' %}
-                            <p><a href="{{ object.ynr_link }}">{% trans "Add some candidates at our candidate crowd-sourcing site" %}</a></p>
-                        {% endif %}
+                        {% for postelection in object.postelection_set.all %}
```

Comment:
This is going to show the line below for _each_ ballot in the election, isn't it? if `object` is an `Election`, it could have ~30 related `PostElection` objects.

Better to take the first one and find out if the SOPN date has past. It's not true that all ballots in an election will have the same timetable, but it's close enough for this PR I think.